WebRoot/messages/ZmMsg_ar.properties

```diff
@@ -2701,6 +2701,7 @@ presentation = \u0639\u0631\u0636 \u062a\u0642\u062f\u064a\u0645\u064a
 presentationName = \u0627\u0633\u0645 \u0627\u0644\u0639\u0631\u0636 \u0627\u0644\u062a\u0642\u062f\u064a\u0645\u064a
 preview = \u0645\u0639\u0627\u064a\u0646\u0629
 previewDownloadLink = \u064a\u062a\u0639\u0630\u0631 \u0639\u0644\u064a\u0646\u0627 \u0625\u0646\u0634\u0627\u0621 \u0645\u0639\u0627\u064a\u0646\u0629 \u0644\u0644\u0645\u0644\u0641. \u0627\u0646\u0642\u0631 \u0647\u0646\u0627 \u0645\u0646 \u0623\u062c\u0644 <a href="{0}" target="_blank" >\u062a\u0646\u0632\u064a\u0644</a> \u0627\u0644\u0645\u0639\u0627\u064a\u0646\u0629.\u200f
+previewNewTabLink = \u0644\u0623\u0633\u0628\u0627\u0628 \u0623\u0645\u0646\u064a\u0629 \u060c \u064a\u0631\u062c\u0649 <a href="{0}" target="_blank" >\u0627\u0644\u0646\u0642\u0631 \u0647\u0646\u0627</a> \u0644\u0639\u0631\u0636\u0647\u0627 \u0641\u064a \u0639\u0644\u0627\u0645\u0629 \u062a\u0628\u0648\u064a\u0628 \u062c\u062f\u064a\u062f\u0629.
```

Comment:
Translation is missing for some languages (`en_AU`, `en_GB`, etc). Do we need to add it?

Reply:
en_AU and en_GB will fallback to the default labels present in ZmMsg.properties which are in English. Which I think is expected, had a discussion regarding same with @miteshsavani810 as well. Let me know if anything else is missing.